spec/inputs/eventlog_spec.rb

```diff
@@ -1,7 +1,11 @@
 require "logstash/devutils/rspec/spec_helper"
-require 'logstash/inputs/eventlog'
```

Comment:
this is not the right place since the exclusion happens at the rspec tag selection. if a warning is to be printed, it should be at that place with something like "currently excluding X Y  Z for platform A B"